### PR TITLE
Remove txpool add cli command

### PR DESCRIPTION
**Pull request by ZeljkoBenovic**
* remove txpool add cli command as we don't support unsigned transactions anymore
*  remove duplicate restore command

### CODE REVIEW

File: docs/get-started/cli-commands.mdx [hunk 1]
```diff
@@ -499,25 +499,6 @@ Set block production time in seconds. Default: `2`
...
-
-    server --restore backup.dat
-
-  </TabItem>
-</Tabs>
-
-Restore blocks from the specified archive file
-
----
-
```
Comment (zivkovicmilos) [inline docs/get-started/cli-commands.mdx:520]: Why was this removed?

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯 

Please see my comment 🙏

**Review by Aleksao998 — CHANGES_REQUESTED** [review]
Apart from the @zivkovicmilos comment, looks good!

File: docs/get-started/cli-commands.mdx (hunk 1 above)
Comment (ZeljkoBenovic) [reply docs/get-started/cli-commands.mdx:520]: It is a duplicate.
You can check [here](https://edge-docs.polygon.technology/docs/get-started/cli-commands) under the server commands, that we have restore twice
